config/routes.rb

```diff
@@ -8,8 +8,8 @@
   post "/discourse-post-event/events/:id/csv-bulk-invite" => "events#csv_bulk_invite"
   post "/discourse-post-event/events/:id/bulk-invite" => "events#bulk_invite", :format => :json
   post "/discourse-post-event/events/:id/invite" => "events#invite"
-  put "/discourse-post-event/events/:post_id/invitees/:id" => "invitees#update"
-  post "/discourse-post-event/events/:post_id/invitees" => "invitees#create"
+  put "/discourse-post-event/events/:event_id/invitees/:invitee_id" => "invitees#update"
+  post "/discourse-post-event/events/:event_id/invitees" => "invitees#create"
```

Comment:
I probably should have avoided doing it without refactoring all of them, but I did it... No excuse.